config/config.go

```diff
@@ -28,8 +28,6 @@ var (
 	DefaultAuth = Auth{
 		Community:     "public",
 		SecurityLevel: "noAuthNoPriv",
-		AuthProtocol:  "MD5",
```

Comment:
This is a breaking change, users may not have these in their configs currently. I don't see a reason to remove them.

Reply:
Fair enough, I can open another issue/PR to discuss how to handle noAuthNoPriv as currently the default is broken (leads to error in gosnmp for using noAuthNoPriv with Authentication protocol defined).